apps/eqeqeq/.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+  "extends": ["../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "extends": [
+        "plugin:@nrwl/nx/angular",
+        "plugin:@angular-eslint/template/process-inline-templates"
+      ],
+      "rules": {
+        "@angular-eslint/directive-selector": [
+          "error",
+          {
+            "type": "attribute",
+            "prefix": "nxExample",
+            "style": "camelCase"
+          }
+        ],
+        "@angular-eslint/component-selector": [
+          "error",
+          {
+            "type": "element",
+            "prefix": "nx-example",
+            "style": "kebab-case"
+          }
+        ],
+        "@angular-eslint/template/eqeqeq": "off"
```

Comment:
You should add the rule for `*.html` files.

```js
    {
      "files": ["*.html"],
      "extends": ["plugin:@nrwl/nx/angular-template"],
      "rules": {
        "@angular-eslint/template/eqeqeq": "off"
      }
    }
  ]
}
```

It sounds weird but it works both separated template (`*.html`) and inline template (`*.ts`).

Reply:
yes it working now for me